demos/burgers/burgers.py.rst

```diff
@@ -90,7 +90,7 @@ To output the initial conditions, we project them into P1 and pipe
 them into the output file::
 
   outfile = File("burgers.pvd")
-  outfile << project(u, V_out, name="Velocity")
+  outfile.write(project(u, V_out, name="Velocity"))
```

Comment:
I see the original projects as well, so just asking: can't we use `project_output` here?